www/static/js/docs.js

```diff
@@ -40,7 +40,7 @@ $(document).ready(function () {
 
     // Table of Contents
     $('#page-toc').toc({
-        'selectors':         'h1,h2,h3,h4,h5,h6', // elements to use as headings
+        'selectors':         'h1,h2', // elements to use as headings
```

Comment:
@riknoll Ideally, I would have preferred a seperate commit. But this is a one-off case. With the changes to the events page, ToC will be cuttered. So, adding it here.